rosetta-lang/src/main/java/com/regnosys/rosetta/generator/java/expression/ExpressionGenerator.xtend

```diff
@@ -489,7 +482,7 @@ class ExpressionGenerator extends RosettaExpressionSwitch<JavaStatementBuilder,
 
 	private def buildConstraint(RosettaExpression arg, Collection<RAttribute> usedAttributes,
 		Necessity validationType, Context context) {
```

Comment:
How to break this method:
```
type Foo:
  a string (1..1)
  b int (0..1)
  c number (0..1)

func MyFunc:
  inputs:
    foo Foo (1..1)
      [metadata reference]
  output:
    result boolean (1..1)
  set result:
    foo required choice b, c
```
This will result in a static compilation error.

Reply:
fixed